oaknational/Oak-Web-Application
# feat: add stop filters to construct query

## Description

- Adds stop filters to elasticsearch query. This will remove :  a, an, and, are, as, at, be, but, by, for, if, in, ...  etc 


## Issue(s)

Fixes #1210 

## How to test

Search for 'a the and is midsummer nights dream' it should only match 'midsummer nights dream'

1. Go to {cloud link}
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
